FFXIVClientStructs/FFXIV/Client/UI/Agent/AgentFreeCompanyProfile.cs

```diff
@@ -45,7 +45,7 @@ public unsafe partial struct AgentFreeCompanyProfile
         [FieldOffset(0x27A)] public byte Unk27A;
         [FieldOffset(0x27B)] public byte Unk27B;
 
-        [StructLayout(LayoutKind.Explicit, Size = 0x7)]
+        [StructLayout(LayoutKind.Explicit, Size = 0x6)]
```

Comment:
this doesn't seem right if there is a Recruitment byte at offset 6 it needs to be at least 7 big

Reply:
Yeah, something's not right.  The part that I was hitting was simply:
        [FieldOffset(0x064)] public FCProfile Profile; // size 0x7 means next bit is 0x6B...
        [FieldOffset(0x06A)] public GrandCompany GrandCompany;
It looks Recruitment is actually at offset 0x5 not 0x6.  Updated.

Reply:
@pohky let me know if there's anything else you see a problem with.